tests/manage/cluster/cluster_shutdown/test_shutdown.py

```diff
@@ -0,0 +1,105 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import ceph_health_check
+from ocs_ci.ocs import ocp
+from ocs_ci.framework.testlib import tier4, ManageTest
+from ocs_ci.utility import aws
+from ocs_ci.ocs.cluster import CephCluster
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def instances(request):
+    """
+    Get cluster instances
+
+    Returns:
+        list: The cluster instances dictionaries
+    """
+    def finalizer():
+        """
+        Make sure all instances are running
+        """
+        aws.start_instances(instances)
+
+    request.addfinalizer(finalizer)
+
+    instances = ocp.get_all_nodes()
+    return instances
+
+
+@pytest.fixture()
+def vms():
+    # TODO
```

Comment:
you can bring in the TODO's in next PR other than that it all LGTM.

Reply:
done